boards/dactyl_tracer/README.md

```diff
@@ -0,0 +1,28 @@
+# Dactyl Tracer
+
+![TopElev](https://i.imgur.com/ReCBppEh.jpeg)
+
+A split body, concave shaped key well, columnar keyboard, with case halves designed similarly to the [Dactyl CC](/boards/dactyl/README.md#dactyl-cc--ergo-s-1).  
+The 'Q' key is vertically offset, from 'A' key, to allow more comfortable actuation using ring finger.  
+Key arrangement bears slight resemblance to the [Dactyl ManuForm](/boards/dactyl_manuform/) in that:
+1. Row spacing and placement of two outward most thumb keys are identical
+2. Bottom finger key-well row has two keys  
```

Comment:
```suggestion
1. Row spacing and placement of two outward most thumb keys are identical and
2. bottom finger key-well row has two keys.
```
That's a suggestion, the exact wording and punctuation I leave up to you -- but: if a list reads like a sentence, it should adhere to the syntax of a sentence.